pom.xml

```diff
@@ -62,8 +62,7 @@
         </pluginRepository>
     </pluginRepositories>
     <properties>
-        <jenkins.version>1.642.3</jenkins.version>
-        <jenkins-test-harness.version>2.19</jenkins-test-harness.version>
+        <jenkins.version>2.7.3</jenkins.version>
```

Comment:
Did not seem practical to support Groovy 1.x, since we have to choose a Groovy release to generate `CpsDefaultGroovyMethods` from. Too old, and we do not cover enough methods; too new, and it may not link against an older Groovy core.